yarn-project/acir-simulator/package.json

```diff
@@ -35,7 +35,7 @@
     "@aztec/circuits.js": "workspace:^",
     "@aztec/foundation": "workspace:^",
     "@aztec/types": "workspace:^",
-    "acvm_js": "github:noir-lang/acvm-js-wasm#db/init-sim-backend",
+    "acvm_js": "github:noir-lang/acvm-js-wasm#arv/0.22+init-pedersen",
```

Comment:
Updated the dependency from 0.21 + init pedersen to 0.22 + init pedersen. Init pedersen will eventually be released as part of 0.23, let's hope that lands soon!